sources/packages/web/src/types/AppRoutes.ts

```diff
@@ -63,6 +63,8 @@ export enum AppRoutes {
   Reports = "reports",
   ApplicationExceptionsApproval = "application-exceptions/:exceptionId",
   Exceptions = "exceptions",
+  OfferingChangeRequests = "institution/offering/change-requests",
+  ViewOfferingChangeRequest = "institution/offering/view-change-request/:offeringId/program/:programId",
```

Comment:
If not is this PR, we should remove the ability to provide the `programId` in the URL it could open a door for a user to change the parameters in the URL.